t/40-script_load_templates.t

```diff
@@ -64,6 +74,35 @@ $filename = 't/data/40-templates.json';
 $args     = "--host $host --apikey $apikey --apisecret $apisecret $filename";
 $expected = qr/Bad Request: Erroneous parameters \(template missing\)/;
 test_once $args, $expected, 'YAML template is mandatory (JSON)', 255, 'failed to load templates without YAML (JSON)';
+
+my $fh;
+($fh, $filename) = tempfile(UNLINK => 1, SUFFIX => '.json');
+$args     = "--host $host --apikey $apikey --apisecret $apisecret --json > $filename";
+$expected = qr/^$/;
+dump_templates $args, $expected, 'dumped fixtures';
+kill TERM => $pid;
+$mojoport = Mojo::IOLoop::Server->generate_port;
+$host     = "localhost:$mojoport";
+$pid      = OpenQA::Test::Utils::create_webapi(
```

Comment:
I assume the previously started web UI can not be used because the fixtures interfere with that test. Have you checked whether it would be possible to get rid of these fixtures instead of starting a new web UI? Maybe the previous test doesn't even need fixtures at all and we can save the time to initialize them completely?

Note that it is also totally possible to initialize (and access) the database/schema in the main test process and to just reuse that in the web UI sub-process (see `t/05-scheduler-full.t`).

Reply:
Yes. The reason was that the fixtures interfere. Another clean database seemed more robust than trying to clean-up by hand... and I didn't see any existing tests/helpers to do that... I'm looking into it, though